src/renderers/shaders/ShaderLib/meshphong_vert.glsl

```diff
@@ -40,9 +40,9 @@ void main() {
 #endif
 
 	#include <begin_vertex>
-	#include <displacementmap_vertex>
 	#include <morphtarget_vertex>
 	#include <skinning_vertex>
+	#include <displacementmap_vertex>
```

Comment:
To be as precise as possible, you are displacing along the "morph-skinning-adjusted" normal in object space.

Ignoring skinning for the moment, morph normals do not have to be specified -- they are optional. One can specify morph targets only.  In that case, the "adjusted" normal is the original normal.

It seems to me that in that case, displacement should be first. That would be problematic, unless you can demonstrate that order does not matter...